packages/e2e-tests/tests/user-invitation/invitation.spec.js

```diff
@@ -0,0 +1,64 @@
+const { AdminCreateUserPage } = require('../../fixtures/admin/create-user-page');
+const { publicTest, expect } = require('../../fixtures/index');
+const { subDays } = require('date-fns');
+const { client } = require('../../fixtures/postgres-client-config');
+const { v4: uuidv4 } = require('uuid');
+
+publicTest.describe('Accept invitation page', () => {
+  publicTest.beforeAll(async () => {
+    await client.connect();
+  });
+
+  publicTest.afterAll(async () => {
+    await client.end();
+  });
+
+  publicTest('should not be able to set the password if token is empty', async ({ acceptInvitationPage }) => {
+    await acceptInvitationPage.open('');
+    await acceptInvitationPage.excpectSubmitButtonToBeDisabled();
+    await acceptInvitationPage.fillPasswordField('something');
+    await acceptInvitationPage.excpectSubmitButtonToBeDisabled();
+  });
+
+  publicTest('should not be able to set the password if token is expired', async ({ acceptInvitationPage, page }) => {
+    const dateToday = new Date().toISOString().replace('T', ' ');
+    const expiredTokenDate = subDays(dateToday, 3).toISOString().replace('T', ' ');
+    const expiredToken = uuidv4();
```

Comment:
The token doesn't have to be a UUID at all. So, I'd say we remove the package and create a random/hardcoded string for the token.